### ISSUE TITLE
Question/Feature Request: Support Web Content Accessibility Guidelines (WCAG)

### ISSUE THREAD

**Issue by theroch** [fabianwennink/IconCaptcha-Plugin-jQuery-PHP#19]
Is there a way to support Web Content Accessibility Guidelines ([WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/))  with this captcha?
WCAG documents explain how to make web content more accessible to people with disabilities.

Maybe it is possible to support screen readers?
What have to be modified in the code to support this?

Or is it possible to provide some audio streams as alternative to the images?
But how can the solution be entered in the formular?

Maybe someone else have a better idea to support WCAG?

**Comment by fabianwennink** [comment]
Hi @theroch! At the moment, IconCaptcha has no accessibility support built-in, and no alternative challenge is available for people with visual impairments. I am aware that this is not great and have looked into providing an alternative audio challenge for better accessibility - with no success yet. I'm not sure if/when an alternative challenge will be available, as my knowledge about audio files/steams is basically zero. Note that the next major release will provide better ARIA support, so that's at least a step in the right direction.

If anyone has an idea on how to implement said feature or has knowledge about working with audio files/streams in PHP, help/feedback would be appreciated.
